packages/docs/src/hooks/usePaddle.tsx

```diff
@@ -1,10 +1,41 @@
 import { useEffect, useState } from "react";
 import { initializePaddle, InitializePaddleOptions, Paddle } from "@paddle/paddle-js";
+import { track } from "@vercel/analytics";
+
+const eventCallback = (event) => {
+  const { name, data } = event;
+  const { items } = data;
+  const [item] = items;
+  const payload = { name: item.price_name, interval: item.billing_cycle.interval };
```

Comment:
I assume name is something like "business" "personal", if so could you rename this to something more descriptive like "pricingName", "pricingType", "subscribtionType" ?

Reply:
Fixed in [bf24b0a](https://github.com/software-mansion/radon-ide/pull/828/commits/bf24b0af8906931c31d66bc78e7a4a000aecd251)

Answering your question: I don't think so.

It tracks individual page views by hashing an incoming request on the server. Tracking specific user actions would require using more advanced analytics with cookies. We would then need to show a cookie banner etc.

https://vercel.com/docs/analytics/privacy-policy#visitor-identification-and-data-storage